### PR TITLE
1047 blog post 5.2.4

### PR DESCRIPTION
**Pull request by tst-nfarah**
This address #1047

### CODE REVIEW

File: docs/_posts/2018-10-08-524-release.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**C++98 no longer supported**. Building open GEE using C++98 standard is deprecated and is no longer supported. C++11 syntax can be used.
+
+**New build parameter to redirect the build output**. Build output can now be optionally directed to a folder of your choosing instead of the current hard coded defaults. See earth_enterprise/BUILD.md in source code for more details.
+
+**New experimental build cache folder parameter to the build**. This is an experimental option to potentially speed up builds. More work remains to be done before using this option in production builds. Use of this option is not recommended except for giving feedback and/or helping to address issues with Open GEE's build process using scons cache. See earth_enterprise/BUILD.md in source code for more details.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.4-2.final). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160004.html).
+ 
+Our thanks goe out to all of the contributors who helped make this release possible! The next release, Open GEE 5.2.5, is already underway!
```
Comment (tst-blaird) [inline docs/_posts/2018-10-08-524-release.md]: Typo:  "goe" should be "goes".

File: docs/_posts/2018-10-08-524-release.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**C++98 no longer supported**. Building open GEE using C++98 standard is deprecated and is no longer supported. C++11 syntax can be used.
+
+**New build parameter to redirect the build output**. Build output can now be optionally directed to a folder of your choosing instead of the current hard coded defaults. See earth_enterprise/BUILD.md in source code for more details.
+
+**New experimental build cache folder parameter to the build**. This is an experimental option to potentially speed up builds. More work remains to be done before using this option in production builds. Use of this option is not recommended except for giving feedback and/or helping to address issues with Open GEE's build process using scons cache. See earth_enterprise/BUILD.md in source code for more details.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.4-2.final). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160004.html).
+ 
+Our thanks goes out to all of the contributors who helped make this release possible! The next release, Open GEE 5.2.5, is already underway!
```
Comment (tst-swanson) [inline docs/_posts/2018-10-08-524-release.md]: Our thanks *go* out

File: docs/_posts/2018-10-08-524-release.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ 
+Enhancements include:
+
+**Progress indicators during gesystemmanager processing**. Added progress log messages to gesystemmanager while assets are being built.
+
+**New "Save and Build" file menu option**. Fusion UI now includes a new file menu option that allows users to save and build assets with one click, instead of clicking the save option and then the build option immediately after.
+
+**C++98 no longer supported**. Building open GEE using C++98 standard is deprecated and is no longer supported. C++11 syntax can be used.
+
+**New build parameter to redirect the build output**. Build output can now be optionally directed to a folder of your choosing instead of the current hard coded defaults. See earth_enterprise/BUILD.md in source code for more details.
```
Comment (tst-swanson) [inline docs/_posts/2018-10-08-524-release.md]: *hard-coded*

File: docs/_data/versions.yml [hunk 4]
```diff
@@ -5,6 +5,8 @@
 - version: 5.2.2
   stability: stable
 - version: 5.2.3
+  stability: stable
+- version: 5.2.4
   stability: latest stable
 - version: 5.2.4
```
[the remark this answers is not in the document]
Comment (tst-sbayard) [reply docs/_data/versions.yml]: Looks like it

**Comment by tst-kfreiheit** [comment]
The git 1.8.3.4 minimum version is baked into the code for 5.2.4 -- the version string is calculated using calls to "git describe" with the "--first-parent" parameter, which was introduced in 1.8.3.4.

It is required to **BUILD** the project, but is not required to download, install, and run the .zip.

A patch on the master branch is forthcoming to re-enable builds with earlier versions of git (same as before), but with the potential to have incorrect version string calculated for builds when using the older versions of git.

I thought this would be worth noting in the same way that C++ deprecation was noted.

File: docs/_posts/2018-10-08-524-release.md (hunk 2 above)
Comment (tst-nfarah) [reply docs/_posts/2018-10-08-524-release.md]: ok

**Comment by tst-mbiancalana** [comment]
@tst-kfreiheit The git version we need is actually 1.8.4 or higher. I had originally thought (based on commit timestamps, I think) that 1.8.3.4 was the correct version, but the `--first-parent` flag was not merged into a release until 1.8.4.

**Comment by tst-nfarah** [comment]
Opening new PR #1060 to fix changes mismatch with target branch

**Pull request closed without merging**